Kernel/Random.cpp

```diff
@@ -69,6 +69,8 @@ UNMAP_AFTER_INIT KernelRng::KernelRng()
             current_time += 0x40b2u;
         }
     }
+#elif ARCH(AARCH64)
```

Comment:
let's not do this until we have an RNG source for aarch64 to make the extra ifdef branch meaningful.

Reply:
i was working on it but it seems itʼs not really a thing for raspberry processor… 😒
(it has some additional hardware but i donʼt know yet how to access it)

iʼll remove that then. 🙇🏻‍♂️